package.json

```diff
@@ -13,6 +13,9 @@
     "build:wasm": "cd wasm && yarn build",
     "build:sdk": "cd sdk && yarn build",
     "build:all": "yarn build:wasm && yarn build:sdk",
+    "build:website": "yarn && yarn build:all",
```

Comment:
This wouldn't build the website I think you maybe wanted:

```cd website && yarn build```

Reply:
I was mainly just thinking to have a command to pre-build the website since it might be difficult for newcomers to know the exact steps. We can update this in the documentation though.

Reply:
Removed!